concent_api/core/views.py

```diff
@@ -610,6 +704,16 @@ def handle_send_force_subtask_results(request, client_message: message.concents.
             provider_public_key  = client_public_key,
             requestor_public_key = other_party_public_key,
         )
+        store_or_update_subtask(
```

Comment:
@cameel Please check if `next_deadline` is defined correctly and if correct related messages are stored.